index.html

```diff
@@ -1,48 +1,316 @@
+            don’t hestiate to contact me.</p>
 
+            <ul class="connect" >
+                <li id="twitter" class="media"><a href="#"><img alt="twitter" src="img/twitter.png"></a></li>
+                <li class="media"><a href="#"><img alt="linkedin" src="img/Linkedin.png"></a></li>
+                <li class="media"><a href="#"><img alt="medium" src="img/medium.png"></a></li>
+                <li class="media"><a href="#"><img alt="github" src="img/github.png"></a></li>
+                <li class="media"><a href="#"><img alt="angellist" src="img/angellist.png"></a></li>
+            </ul>
+
+
+        </section>
+
+    
+    
+    </main>
 
- </section>
- </main>
  
     
-</body>
+        <header>
+
+            <div><button  class="scroll-down" type="button">
+                <i class="fa fa-angle-down fa-2x"></i>
+            </button></div>
+
+            <div class="recent-works"><h1>My Recent Works</h1></div>
+            <div class="indicator">
+                <img alt="indcator" src="img/Indicator.png">
+            </div>       
+
+        </header>
+    <div class="wrapper">
+
+        <div>
+            <section id="works1" class="container3 cnt">
+
+
+                <div class="item3"><img alt="work1" src="img/work1.png"></div>
+
+                
+                            
+                <div class="item3"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item3 btn-group">
+                    <ul id="btn1">
+                    <li><button class="languages"> Ruby on rails</button></li>
+                    <li><button class="languages"> CSS</button></li>
+                    <li><button class="languages"> JavaScript</button></li> 
+                    <li><button class="languages"> HTML</button></li>
+                    </ul>
+                </div>
+
+                
+                    <div class="item3">
+                        <button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works2" class="container4">
+
+
+                <div class="item4"><img alt="work2" src="img/work2.png"></div>
+
+                
+                            
+                <div class="item4"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item4 btn-group">
+                    <ul id="btn2">
+                        <li><button class="languages"> Ruby on rails</button></li>
+                        <li><button class="languages"> CSS</button></li>
+                        <li><button class="languages"> JavaScript</button></li> 
+                        <li><button class="languages"> HTML</button></li>
+                        </ul>
+                </div>
+
+                
+                <div class="item4"><button class="see-project">See project</button></div>
+                
+                
+
+
+
```

Comment:
- [ ] **OPTIONAL:** Please pay attention to README.md Acknowledgements section, kindly add something special to this project, or if you don't want you can remove it. 
- [ ] **OPTIONAL:** Also I highly recommend to add live demo link with the help of GitHub pages, because not everybody knows coding and opening live version on local machine, it easier to see what you have done via live link for recruiters.